### PR TITLE
SIG docs Hindi team

### PR DESCRIPTION
**Pull request by mittalyashu**
This PR should create a Hindi team.

### CODE REVIEW

**Comment by nikhita** [comment]
@mittalyashu Once the PR is rebased, it should be good to go. :+1: 

/assign @zacharysarah

**Comment by mittalyashu** [comment]
Should I squash the commits too?

File: sigs.yaml [hunk 1]
```diff
@@ -975,6 +975,67 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes/kubernetes-anywhere/master/OWNERS
   - name: kubespray
     contact:
+      slack: sig-docs
```
Comment (nikhita) [inline sigs.yaml]: @mittalyashu these changes don't look correct (they are under kubespray).

Can you fix this and squash the commits?

**Comment by zacharysarah** [comment]
@mittalyashu 👋 Please feel free to reopen this PR when you're ready to continue with feedback.

/close

**Comment by mittalyashu** [comment]
Oh! Apologies for the delay in response 🤔.

/open

**Comment by mittalyashu** [comment]
@zacharysarah I am unable to open the PR again.

File: sigs.yaml (hunk 1 above)
Comment (mittalyashu) [reply sigs.yaml]: > @mittalyashu these changes don't look correct (they are under kubespray).

That's weird, let me fix that.

> Can you fix this and squash the commits?

After the final review, I will squash the commits.

Comment (mittalyashu) [reply sigs.yaml]: I have just checked the history of the `sigs.yaml` file and somethings fishy going on there 😅.

**Comment by cblecker** [comment]
/hold

This PR needs to be rebased against master, as it's bringing in changes like sig-gcp stuff.

**Comment by nikhita** [comment]
@mittalyashu :wave: can you rebase the PR? looks good otherwise :)

**Comment by mittalyashu** [comment]
> @mittalyashu 👋 can you rebase the PR? looks good otherwise :)

But I am still trying to fix this issue: https://github.com/kubernetes/community/pull/3711#discussion_r288557914

Maybe someone else might have fixed it?

**Comment by zacharysarah** [comment]
@mittalyashu I hope you continue this PR! However, please `/reopen` this PR only when you're willing to continue to completion.

/close

**Comment by nikhita** [comment]
@zacharysarah This looks good to go, PTAL.

**Comment by zacharysarah** [comment]
@mittalyashu Thanks for your patience, I was on vacation. 

/lgtm
/approve

**Pull request merged**